site/gatsby-site/src/components/submissions/SubmissionForm.js

```diff
@@ -402,7 +411,17 @@ const SubmissionForm = () => {
         )}
 
         <details className="mt-3">
-          <summary data-cy="extra-fields">{t('Tell us more...')}</summary>
+          <summary data-cy="extra-fields">
+            {t('Tell us more...')}
+            {(errors['description'] ||
+              errors['developers'] ||
+              errors['deployers'] ||
+              errors['harmed_parties']) && (
+              <div className="text-red-500 pl-4">
+                <Trans ns="validation">Some data is missing.</Trans>
```

Comment:
Of course, I'm guilty of this, but it is funny how we devs are bad at giving helpful error messages.

Reply:
Yeah, I tried to let the user know that something is missing in a generic way on the `Tell me more...` section.
And if the user expands the section the explicit error messages are displayed.